modules/dhcp_common/dhcp_common.rb

```diff
@@ -19,7 +19,7 @@ module Proxy::DHCP
     :install_server_name   => {:code => 11, :kind => "String"    }, # mediahost
     :install_path          => {:code => 12, :kind => "String"    }, # /vol/solgi_5.10/sol10_hw0910
     :sysid_server_path     => {:code => 13, :kind => "String"    }, # 192.168.216.241:/vol/jumpstart/sysidcfg/sysidcfg_primary
-    :jumpstart_server_path => {:code => 14, :kind => "String"    }  # 192.168.216.241:/vol/jumpstart
+    :jumpstart_server_path => {:code => 14, :kind => "String"    },  # 192.168.216.241:/vol/jumpstart
```

Comment:
Perhaps we should keep the comments aligned
```suggestion
    :jumpstart_server_path => {:code => 14, :kind => "String"    }, # 192.168.216.241:/vol/jumpstart
```

Reply:
Fixed.